src/i18n.ts

```diff
@@ -9,7 +9,7 @@ const resources = {
       "Find citizen": "Find citizen",
       Submit: "Submit",
       "Citizen profile": "Citizen profile",
-      "Payment methods": "Payment methods",
+      "Payment methods": "Payment methods Cashback enabled",
```

Comment:
Could be enabled or not.
```suggestion
      "Payment methods": "Payment methods Cashback",
```